docker/circleci/Dockerfile

```diff
@@ -1,40 +1,16 @@
-FROM circleci/golang:1.15-node-browsers
+FROM cimg/go:1.15.6-browsers
```

Comment:
The old base images are deprecated in favor of `cimg`: https://circleci.com/docs/2.0/circleci-images/#next-generation-convenience-images